onvm/onvm_nflib/onvm_nflib.c

```diff
@@ -727,7 +727,14 @@ onvm_nflib_send_msg_to_nf(uint16_t dest, void *msg_data) {
         msg->msg_type = MSG_FROM_NF;
         msg->msg_data = msg_data;
 
-        return rte_ring_enqueue(nfs[dest].msg_q, (void*)msg);
+        uint16_t instance_id = onvm_sc_service_to_nf_map(dest, NULL);
+        ret = rte_ring_enqueue(nfs[instance_id].msg_q, (void*)msg);
+        if (ret != 0) {
+                RTE_LOG(ERR, APP, "Destination NF ring is full! Unable to enqueue msg to ring\n");
```

Comment:
```suggestion
                RTE_LOG(WARNING, APP, "Destination NF ring is full! Unable to enqueue msg to ring\n");
```
Since this isn't technically an error (it just means the system is overloaded), let's make this a warning instead of an error. That way if we are trying to run high speed performance tests and disable all logging except for ERRORs, this message won't appear.  Every print statement can slow down the system.